packages/image_picker/image_picker_windows/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: image_picker_windows
 description: Windows platform implementation of image_picker
 repository: https://github.com/flutter/packages/tree/main/packages/image_picker/image_picker_windows
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+image_picker%22
-version: 0.2.0
+version: 0.1.1
```

Comment:
0.2.1?

Reply:
bad merge conflict handling on my part. OR... 1.1 may in fact be higher than 2.0.